### PR TITLE
I1640 proper labeling open shift console upstream backup

### PR DESCRIPTION
**Pull request by MichalMaler**
Adding a new attribute {prod2}

Updating labels in OpenShift console procedures

Using principles of the mighty minimalism

### CODE REVIEW

File: src/main/pages/che-7/overview/proc_finding-che-cluster-url-in-openshift-4-web-console.adoc [hunk 1]
```diff
@@ -3,27 +3,24 @@
 
 This section descibes how to find the {prod-short} cluster URL in OpenShift 4 web console.
 
-[discrete]
-== Prerequisites
+.Prerequisites
 
-* A running {prod} cluster. 
+* A running {prod} cluster.
```
Comment (rkratky) [inline src/main/pages/che-7/overview/proc_finding-che-cluster-url-in-openshift-4-web-console.adoc]: "Eclipse Che cluster"?

File: src/main/pages/che-7/overview/proc_finding-che-cluster-url-in-openshift-4-web-console.adoc [hunk 2]
```diff
@@ -3,27 +3,24 @@
...
 
-[discrete]
-== Procedure
+.Procedure
 
 . Open the OpenShift web console.
 
 . In the left panel, navigate to the *Operators* -> *Installed Operators* section.
 
 . Click on the *{prod}* Operator tile.
```
Comment (rkratky) [inline src/main/pages/che-7/overview/proc_finding-che-cluster-url-in-openshift-4-web-console.adoc]:
```suggestion
. Click the *{prod}* Operator tile.
```
Never "click on".

File: src/main/pages/che-7/overview/proc_finding-che-cluster-url-in-openshift-4-web-console.adoc [hunk 3]
```diff
@@ -3,27 +3,24 @@
...
 
-. Click on *eclipse-che* in the table.
-
-. Navigate to the *Overview* tab.
+. Click on *{prod-checluster}* `CheCluster` in the table.
```
Comment (rkratky) [inline src/main/pages/che-7/overview/proc_finding-che-cluster-url-in-openshift-4-web-console.adoc]:
```suggestion
. Click *{prod-checluster}* `CheCluster` in the table.
```

File: src/main/pages/che-7/overview/proc_viewing-the-state-of-the-che-cluster-deployment-in-openshift-4-web-console.adoc [hunk 4]
```diff
@@ -3,35 +3,33 @@
...
 This section describes how to view the state of the {prod-short} cluster deployment in OpenShift 4 web console.
 
-[discrete]
-== Prerequisites
+.Prerequisites
 
 * An administrator account on a running instance of OpenShift 4.
 
-* A {prod-short} cluster is deployed on this instance of OpenShift 4. 
+* A {prod-short} cluster is deployed on this instance of OpenShift 4.
```
Comment (rkratky) [inline src/main/pages/che-7/overview/proc_viewing-the-state-of-the-che-cluster-deployment-in-openshift-4-web-console.adoc]: Again... "Che cluster". Am I missing something, or is this a mistake? I believe there usually is an OpenShift/K8s cluster with an instance of Che...

File: src/main/pages/che-7/overview/proc_viewing-the-state-of-the-che-cluster-deployment-in-openshift-4-web-console.adoc [hunk 5]
```diff
@@ -3,35 +3,33 @@
...
 
 . Click *{prod}* in the list of installed operators.
 
-. Navigate to the *{prod} Cluster* tab.
+. Navigate to the *{prod2} Cluster* tab.
 
-. Click on the *eclipse-che* cluster that is listed in the table.
-
-. Navigate to the *Overview* tab.
+. Click on *{prod-checluster}* `CheCluster` in the table.
```
Comment (rkratky) [inline src/main/pages/che-7/overview/proc_viewing-the-state-of-the-che-cluster-deployment-in-openshift-4-web-console.adoc]:
```suggestion
. Click *{prod-checluster}* `CheCluster` in the table.
```

File: src/main/pages/che-7/overview/proc_finding-che-cluster-url-in-openshift-4-web-console.adoc [hunk 6]
```diff
@@ -3,27 +3,24 @@
 
 This section descibes how to find the {prod-short} cluster URL in OpenShift 4 web console.
```
Comment (rkratky) [inline src/main/pages/che-7/overview/proc_finding-che-cluster-url-in-openshift-4-web-console.adoc]: I'm afraid this whole section is operating with incorrect terminology.

* OpenShift/K8s *cluster*
* Eclipse Che *installation* or *instance* or *deployment* -- but not Eclipse Che *cluster*

Unless we're using Eclipse Che to specify an OpenShift cluster -- as in *Che cluster* (the OpenShift cluster in which Che is deployed).

**Review by rkratky — CHANGES_REQUESTED** [review]

File: src/main/pages/che-7/overview/proc_viewing-the-state-of-the-che-cluster-deployment-in-openshift-4-web-console.adoc (hunk 4 above)
Comment (ScrewTSW) [reply src/main/pages/che-7/overview/proc_viewing-the-state-of-the-che-cluster-deployment-in-openshift-4-web-console.adoc]: Hello. I believe this is Che Cluster deployment as whole (including Postgress, Keycloak, devfile and plugin registry and che deployment itself).

It is possible that I'm mistaken, but I think the line is correct.

**Review by rkratky — APPROVED** [review]
Thanks!

**Pull request merged**